### PR TITLE
-  Create Neumorphic controls

### PR DESCRIPTION
**Pull request by mobile-simformsolutions**
- Create a Neumorphic view, button, and text filed.

### CODE REVIEW

File: Example/SSNeumorphicView/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  SSNeumorphicView
+//
+//  Created by Ankit panchotiya on 07/06/2020.
```
Comment (vishalpatel-simformsolutions) [inline Example/SSNeumorphicView/AppDelegate.swift:5]: Change name to Simform Solutions.

Comment (vishalpatel-simformsolutions) [reply Example/SSNeumorphicView/AppDelegate.swift:5]: In all files.

File: SSNeumorphicView.podspec [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#   * Try to keep it short, snappy and to the point.
+#   * Write the description between the DESC delimiters below.
+#   * Finally, don't worry about the indent, CocoaPods strips it!
+
+  s.description      = 'SSNeumorphicView is to build a Neumorphism view, button, and text filed in iOS.'
+
+  s.homepage         = 'https://github.com/simformsolutions/SSNeumorphicView.git'
+  # s.screenshots     = 'www.example.com/screenshots_1', 'www.example.com/screenshots_2'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'Ankit panchotiya' => 'ankitpanchotiya@simformsolutions.com' }
```
Comment (vishalpatel-simformsolutions) [inline SSNeumorphicView.podspec:25]: Author should be simform solutions.

File: Example/SSNeumorphicView/SSBaseButton.swift [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+import Foundation
+import UIKit
+
+extension UIButton {
+    
+    static func barButton(image: UIImage) -> UIButton {
+        let button = UIButton(type: .custom)
+        button.tintColor = .white
+        button.frame = CGRect(x: 0, y: 0, width: 55, height: 55)
```
Comment (vishalpatel-simformsolutions) [inline Example/SSNeumorphicView/SSBaseButton.swift:17]: height and width should be dynamic

Comment (vishalpatel-simformsolutions) [reply Example/SSNeumorphicView/SSBaseButton.swift:17]: tint colour should be dynamic

File: Example/SSNeumorphicView/SSBaseButton.swift [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        button.setImage(image, for: .normal)
+        button.imageEdgeInsets = UIEdgeInsets(top: 11.5, left: 9.5, bottom: 11.5, right: 9.5)
```
Comment (vishalpatel-simformsolutions) [inline Example/SSNeumorphicView/SSBaseButton.swift:19]: edge insets should be dynamic

**Comment by mobile-simformsolutions** [comment]
Create Easy to use Library for Neumorphic UI component.

**Pull request merged**